src/ci/bin/testCentaurAws.sh

```diff
@@ -34,6 +34,8 @@ cromwell::build::run_centaur \
     -e abort.scheduled_abort \
     -e relative_output_paths \
     -e relative_output_paths_colliding \
-    -e standard_output_paths_colliding_prevented
+    -e standard_output_paths_colliding_prevented \
+    -e space \
+    -e scatter
```

Comment:
Is also disabling the `scatter` test deliberate (presumably so)?

Reply:
Yes, it's also failing for unknown reason. I mentioned that in a comment: https://broadworkbench.atlassian.net/browse/BA-6152?focusedCommentId=25524&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-25524